### PR TITLE
RUN-5568 BrowserView ContentNavigation

### PR DESCRIPTION
**Pull request by aziz512**
#### Description of Change
Adds content navigation filtering for BrowserViews through `contentNavigation` option. Usage is identical to the option in Window

Tests:
https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5d8e665336089855657ecd76

#### Checklist
- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] automated tests are [changed or added](https://testing-dashboard.openfin.co/#/app/dashboard)
- [x] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [x] PR release notes describe the change in a way relevant to app-developers
- [x] Link to new tests added
- [x] PR has assigned reviewers

#### Release Notes
Notes: Added BrowserView `contentNavigation` option analogous to Window

@datamadic @pbaize @MichaelMCoates @rdepena

### CODE REVIEW

**Comment by aziz512** [comment]
Just figured out why it fails to build -- I made a change to a shape in the js-adapter.
Here's the PR for it https://github.com/HadoukenIO/js-adapter/pull/359 . Once that gets merged, we can get back to this PR.

**Comment by pbaize** [comment]
Do we need to document these events in the js-adapter?

**Comment by aziz512** [comment]
> Do we need to document these events in the js-adapter?

@pbaize 
Added docs

File: src/shapes.ts [hunk 1]
```diff
@@ -103,6 +103,11 @@ export interface Window {
     openfinWindow: OpenFinWindow|null;
     parentId?: number;
 }
+
+export interface BrowserView {
```
Comment (pbaize) [inline src/shapes.ts]: Let's add this to OfView rather than here. Extending the electron shapes causes headaches

**Review by pbaize — CHANGES_REQUESTED** [review]
Looks good. See the comment on the BrowserView shape. It would also be good not to have formatting changes in this pr

File: src/shapes.ts (hunk 1 above)
Comment (aziz512) [reply src/shapes.ts]: That shape was not extending the electron shape, it was just used to shape the options.
I renamed it for clarity in the following commit.

**Pull request closed without merging**